ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -82,6 +82,7 @@ CREATE TABLE party_entries (
     typ VARCHAR NOT NULL,
     rejection_reason VARCHAR,
     is_local BOOLEAN,
+    party_id INTEGER,
```

Comment:
is it on purpose to keep the existing `party` column?

Reply:
yes, although I was also a bit uncertain. My arguments where:
* It does not matter much (relatively small table)
* Better for manual DB checking
* In this case it is actually the source of truth which we would intern, and in other cases it is usage
* This we only need for pruning, so if we would not need this for pruning for some reason, it would be easier to track back

Reply:
I second the decision to keep the source of truth table complete.